Documentation/data-cleanup/dashboard_resource_helper.py

```
@@ -0,0 +1,147 @@
+        self.collection_cache = None
+        self.last_cache_time = 0
+        self.cache_ttl = 1800
+        
+    def get_session_token(self):
+        if self.session_token:
+            return self.session_token
+
+        url = f"{self.metabase_url}/session"
+        payload = {"username": self.username, "password": self.password}
+
+        resp = requests.post(url, json=payload)
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)